### DISCUSSION TITLE
Vim Key Bindings Mode for Editing

### DISCUSSION THREAD

**Discussion by GrimnebulinAhab** [TabularEditor/TabularEditor3#972] [Ideas]
Any love for vim? :smiley: In all seriousness, it is a much more efficient editing experience for those who choose to use it.

**Comment by mlonsk** [comment]
I am not sure I follow, it is much more efficient for what compared to? :)

**Reply by GrimnebulinAhab** [reply]
Sorry for being vague. Vim editing mode is a very fast way of changing code compared to conventional keyboard + mouse operation. TE has some fantastic productivity features like regular expressions for find and replace. A Vim editing mode would be a logical extension of that IMO. Thank you for taking the time to reply!
